depends/README.md

```diff
@@ -121,6 +121,8 @@ build script logic) are searched for among the host system packages using
 <dd>(EXPERTS ONLY) When cross-compiling for macOS, use Clang found in the
 system's <code>$PATH</code> rather than the default prebuilt release of Clang
 from llvm.org. Clang 8 or later is required.</dd>
+<dt>LOG</dt>
+<dd>Use file-based logging for individual packages</dd>
```

Comment:
Could you add more details? For example, where would we expect these log files to appear?

Reply:
If wording in the updated OP is ok, I'll push it into the docs.